### ISSUE TITLE
Option to un-zscore to convert to PSC

### ISSUE THREAD

**Issue by gjheij** [lukassnoek/pybest#10]
It would be a great option to un-zscore the data after denoising so we can convert it to percent-signal change. I haven't found a good solution to this yet; z-scoring occurs during the preprocessing step, so one would have to save out the standard deviation and mean in the run-specific config-dictionary. Question is, would the denoising have access to this information after the denoising stage.

**Comment by lukassnoek** [comment]
Yeah saving the mean/sd would be the only option. An alternative would be to add an option to use PSC instead of zscoring as a normalization method. (Right?)
